### PR TITLE
Version 2.0.4 dev -> rc

### PR DESCRIPTION
**Pull request by bleehu**
We've done all of this work, now let's propagate it down from Dev to RC!

Looks like there are some conflicts. Likely stuff that wasn't ready to be released from dev to RC. I'll iron those out before we merge.

### CODE REVIEW

File: play/PlayerRules/CharacterCreation/02_Classes.txt [hunk 1]
```diff
@@ -267,7 +275,8 @@ V
 The Trail becomes bright against a monochrome background of edged shades of 
 grey. +75% to a tracking check, or maximum 2 turns to pass.
 ----------------------
-Highlight Target:
+Highlight Target: (Perception checks do not include checks to hit target. But by using a perception check for an 
+action to see where the target is and passing the target''s Cover DC+25 the player may recieve a +5% / +10% accuracy bonus.)
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/02_Classes.txt:279]: Idk what this perception stuff is talking about. I can't remember who wrote this but.. I can spend an action to find the target? Is this just a normal perception check, except I can get a bonus from it? And is this a separate ability besides everything below?

File: play/PlayerRules/CharacterCreation/02_Classes.txt [hunk 2]
```diff
@@ -280,13 +289,15 @@ Expend 10 nanites to paint a target. All damage dealt by allies to this target
...
+"Paladin" - You are a righteous warrior, vetted and trained by one of the 
+
 powerful corporations or some other technologically advanced entity. You seek 
 to uphold the principals which define the group you''ve sworn allegience to. 
 Through this devotion you have been granted access to high tech in order to 
 pursue your goals and missions.
-Passives: Terms and Conditions clause compliance, access to Paladin Spell tree
+Passives: You gain 45 total nanites and each level you gain an additional 5 
+maximum nanites for each level after your first.
+Terms and Conditions clause compliance, access to Paladin Spell tree
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/02_Classes.txt:300]: "Terms and Conditions clause compliance"??? is this a special power? I am not aware of a clause compliance

File: play/PlayerRules/CharacterCreation/06_Skills.txt [hunk 3]
```diff
@@ -1,112 +1,119 @@
...
+bleed damage. Medicine skill checks can be performed even if untrained in 
+medicine, but the user takes a -10%. Medicine packs decrease the DC by 10, 
+increase the amount healed by 30, and negate bleed damage on a failed roll.
+//Int
+Navigate: you can read a map, congratulations. Any time you are lost, or need 
+to decide where to go, this skill will channel your inner survivorman and 
+enable you to have a better idea of where you should face.
+//Dex
+Piloting: You have been trained in the use and operation of many vehicles, and 
+even if you donâ€™t recognize what it is, you can probably figure out how to use
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/06_Skills.txt]: erm what are those special symbols? "donâ€™t"

File: play/PlayerRules/CharacterCreation/06_Skills.txt [hunk 4]
```diff
@@ -1,112 +1,119 @@
...
+//Per
+Sniping: You have a background going hunting with your dad, or maybe you went 
+to marine scout / sniper school. Who cares. If it moves, breathes or thinks, 
+you can hit it. Some weapons require this skill for use, and this skill can be 
+used for accuracy with any gun that has a range beyond 200m.
+//Dex
+Stealth: The art of stealth is a blanket over the most cloak and dagger of 
+actions. Good thing youâ€™re in on the secret.
+//Int
+Survival: Lost in the wilderness? Youâ€™re journey home will be made easier by
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/06_Skills.txt]: more weirdo characters are here, or maybe github just doesn't support that

File: play/README.txt [hunk 5]
```diff
@@ -1,3 +1,15 @@
-Version 2.0.3
+Version 2.0.4
 
-By Taylor Rowland and Mike Hedges
+Creative Director: 	Taylor Rowland
+Producer:		Mike Hedges
+Lead Developer:		Alex Lindahl
+Lead Test:		Stephen Little
```
Comment (1sourcecontrol) [inline play/README.txt]: That extra tab on my name makes the doc look more unprofessional.

File: play/PlayerRules/CharacterCreation/02_Classes.txt [hunk 6]
```diff
@@ -480,18 +491,21 @@ Starting Wealth: $1700+100*Luck + 2000
...
+Pin can only be activated by a pinned target once per Marksman player's turn. Only usable with long-rifles.
 |
 |
 |
 V
 Multi-Pin
-You may pin up to PER targets at a time. +20% to hit a Pinned target.
+You can now Pin a maximum of PER/2 (round down) targets at  time (you still have to fire at a 
+target to pin it). +20% to hit a Pinned target (instead of +10%). The Multi-Pin ability can be 
+activated by a pinned target once, per target, per Marksman player's turn. (I.e., 3 Pinned targets can
```
Comment (1sourcecontrol) [inline play/PlayerRules/CharacterCreation/02_Classes.txt:507]: > The Multi-Pin ability can be activated by a pinned target once, per target, per Marksman player's turn.

I think we can write that more clearly:
" The Multi-Pin ability can be activated by a pinned target, once per target, per Marksman player's turn."
or "one activation per target" vs "once per target"
what do you think?

**Review by 1sourcecontrol — COMMENTED** [review]
couple of questions on some things

File: play/PlayerRules/CharacterCreation/02_Classes.txt (hunk 1 above)
Comment (bleehu) [reply play/PlayerRules/CharacterCreation/02_Classes.txt:279]: @trowl223 ?

File: play/PlayerRules/CharacterCreation/02_Classes.txt (hunk 2 above)
Comment (bleehu) [reply play/PlayerRules/CharacterCreation/02_Classes.txt:300]: @trowl223 ?

File: play/PlayerRules/CharacterCreation/06_Skills.txt (hunk 3 above)
Comment (bleehu) [reply play/PlayerRules/CharacterCreation/06_Skills.txt]: UTF-8 encoding versus ASCII I believe. we should fix it to the lowest common denominator.

File: play/README.txt (hunk 5 above)
Comment (bleehu) [reply play/README.txt]: tab?

Comment (1sourcecontrol) [reply play/README.txt]: The spacing, my name isn't inline with the others

File: play/PlayerRules/CharacterCreation/02_Classes.txt (hunk 2 above)
Comment (trowl223) [reply play/PlayerRules/CharacterCreation/02_Classes.txt:300]: Paladin code of conduct. I'll add an issue

**Comment by bleehu** [comment]
No review for a month. Merging and beginning pull from RC to play.

File: play/PlayerRules/CharacterCreation/02_Classes.txt (hunk 1 above)
Comment (1sourcecontrol) [reply play/PlayerRules/CharacterCreation/02_Classes.txt:279]: @bleehu / @trowl223  Did this ever get clarified?

**Pull request merged**